docs/sqlalchemy.rst

```diff
@@ -154,6 +154,19 @@ Once you have an CrateDB ``engine`` set up, you can create and use an SQLAlchemy
 
     SQLAlchemy has more documentation about this topic on :doc:`sa:orm/session_basics`.
 
+
+.. _cloud-connect:
+
+Connecting to CrateDB Cloud
+...........................
+
+Connecting to `CrateDB Cloud`_ works like this. Please note the ``?ssl=true``
+query parameter at the end of the database URI.
+
+    >>> import sqlalchemy as sa
+    >>> engine = sa.create_engine("crate://admin:iM_n1%0xRzShsapU4U(lDGOj@example.aks1.westeurope.azure.cratedb.net:4200?ssl=true", echo=True)
```

Comment:
```suggestion
    >>> engine = sa.create_engine("crate://admin:pwd@example.aks1.westeurope.azure.cratedb.net:4200?ssl=true", echo=True)
```

should make it a bit easier to read?

Reply:
Thanks. Maybe I will use `<PASSWORD>`, as we do in the Cloud documentation?

```python
client.connect("https://example.aks1.westeurope.azure.cratedb.net:4200", username="admin", password="<PASSWORD>", verify_ssl_cert=True)
```

Reply:
Fixed with 14028867e.